src/nvcore/Debug.cpp

```diff
@@ -607,6 +607,9 @@ namespace
 #    elif NV_CPU_ARM
         ucontext_t * ucp = (ucontext_t *)secret;
         return (void *) ucp->uc_mcontext->__ss.__pc;
+#    elif NV_CPU_AARCH64

```

Comment:
If this is identical to the NV_CPU_ARM why not handle both simultaneously?

Reply:
There isn't a driving reason to have `NV_CPU_AARCH64` separate from `NV_CPU_ARM`. If you think that it would be better to have them together, I can create a new commit that merges them.

Reply:
I would appreciate that! Thanks!